### PR TITLE
feat: implement xlinectl `delete` and `lease` commands

### PR DESCRIPTION
**Pull request by bsbds**
Please briefly answer these questions:

* what problem are you trying to solve? (or if there's no problem, what's the motivation for this change?)

* what changes does this pull request make?

* are there any non-obvious implications of these changes? (does it break compatibility with previous versions, etc)

### CODE REVIEW

File: xlinectl/src/command/lease/keep_alive.rs [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    Client,
+};
+use xlineapi::LeaseKeepAliveResponse;
+
+use crate::utils::printer::Printer;
+
+/// Definition of `keep_alive` command
+pub(super) fn command() -> Command {
+    Command::new("keep_alive")
+        .about("lease keep alive periodically")
```
Comment (iGxnon) [inline xlinectl/src/command/lease/keep_alive.rs:18]: Capital letter "L" ?

File: xlinectl/src/utils/printer.rs [hunk 2]
```diff
@@ -80,6 +168,7 @@ impl SimplePrinter {
 /// Field Printer of common response types
 struct FieldPrinter;
 
+#[allow(dead_code)]
```
Comment (Phoenix500526) [inline xlinectl/src/utils/printer.rs:171]: Please move the this `#[allow(dead_code)]` to the `FieldPrinter::value` method

File: xlinectl/src/command/delete.rs [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+mod tests {
+    use crate::testcase_struct;
+
+    use super::*;
+
+    testcase_struct!(DeleteRangeRequest);
+
+    #[test]
+    fn valid() {
+        let testcases = vec![
```
Comment (Phoenix500526) [inline xlinectl/src/command/delete.rs:69]: The `testcases` is not a word, so better `test_cases`.

File: xline-client/src/types/kv.rs [hunk 4]
```diff
@@ -368,7 +368,7 @@ impl From<RangeRequest> for xlineapi::RangeRequest {
 }
 
 /// Request type for `DeleteRange`
-#[derive(Debug)]
+#[derive(Debug, PartialEq)]
```
Comment (Phoenix500526) [inline xline-client/src/types/kv.rs:371]: What is this `PartialEq` for?

File: xline-client/src/types/lease.rs [hunk 5]
```diff
@@ -44,7 +44,7 @@ impl LeaseKeeper {
 }
 
 /// Request for `LeaseGrant`
-#[derive(Debug)]
+#[derive(Debug, PartialEq)]
```
Comment (Phoenix500526) [inline xline-client/src/types/lease.rs:47]: Same as above

File: xlinectl/src/command/lease/keep_alive.rs (hunk 1 above)
Comment (bsbds) [reply xlinectl/src/command/lease/keep_alive.rs]: done

File: xlinectl/src/utils/printer.rs (hunk 2 above)
Comment (bsbds) [reply xlinectl/src/utils/printer.rs]: done

File: xlinectl/src/command/delete.rs (hunk 3 above)
Comment (bsbds) [reply xlinectl/src/command/delete.rs]: renamed

File: xline-client/src/types/kv.rs (hunk 4 above)
Comment (bsbds) [reply xline-client/src/types/kv.rs:371]: It used in test in xlinectl, it doesn't seems to working to use cfg_attr for conditional comilation for test cfg only .

**Review by Phoenix500526 — APPROVED** [review]
LGTM

**Pull request merged**